docs/master/subscriptions/getting-started.md

```diff
@@ -23,6 +23,43 @@ If you want to use the Pusher driver, you need to install the [Pusher PHP Librar
 
 If you want to use the Laravel Echo driver, you need to set the env `LIGHTHOUSE_BROADCASTER=echo`.
 
+### Subscriptions version
+
+Lighthouse can return channels in two different formats.
+
+If you want to use version 2, set the env `LIGHTHOUSE_SUBSCRIPTION_VERSION=2`.
```

Comment:
This should refer to a config setting in `lighthouse.php`. The config internally can use an env variable.